ArcadiaTeamsBot/Dialogs/RequestsTypeDialog.cs

```diff
@@ -0,0 +1,89 @@
+﻿namespace ArcadiaTeamsBot.Dialogs
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    using ArcadiaTeamsBot.CQRS.Abstractions;
+    using ArcadiaTeamsBot.ServiceDesk.Abstractions.DTOs;
+    using ArcadiaTeamsBot.ServiceDesk.Requests.RequestTypeFactory;
+
+    using MediatR;
+
+    using Microsoft.Bot.Builder;
+    using Microsoft.Bot.Builder.Dialogs;
+    using Microsoft.Bot.Schema;
+
+    public class RequestsTypeDialog : ComponentDialog
+    {
+        private const string Back = "Back";
+        private static IEnumerable<ServiceDeskRequestTypeDTO> requestTypes;
+        private readonly IMediator mediator;
+
+        public RequestsTypeDialog(IMediator mediator, IRequestTypeUIFactory request) : base(nameof(RequestsTypeDialog))
```

Comment:
Please, fix naming. `IRequestTypeUIFactory request` is not a **request**.

Reply:
Deleted because it's redundant on this stage